From cd3559daa4075c0e4e4a3e93e7f8e934f55eac4a Mon Sep 17 00:00:00 2001
From: Huang Jiachai <hjc@rock-chips.com>
Date: Fri, 22 Jan 2016 17:40:54 +0800
Subject: [PATCH] video: rockchip: screen: init extend screen type

if the extend screen type is same to prmry screen type,
the GRF reg config for vop selete will be error;

Change-Id: I6671ae69da175352bd1935254603e434e85900ea
Signed-off-by: Huang Jiachai <hjc@rock-chips.com>
---
 drivers/video/rockchip/screen/rk_screen.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/rockchip/screen/rk_screen.c b/drivers/video/rockchip/screen/rk_screen.c
index b82a8e7218c3..3a4e6c3bb8ac 100644
--- a/drivers/video/rockchip/screen/rk_screen.c
+++ b/drivers/video/rockchip/screen/rk_screen.c
@@ -14,6 +14,7 @@ int rk_fb_get_extern_screen(struct rk_screen *screen)
 	memcpy(screen, rk_screen, sizeof(struct rk_screen));
 	screen->dsp_lut = NULL;
 	screen->cabc_lut = NULL;
+	screen->type = SCREEN_NULL;
 
 	return 0;
 }
-- 
2.35.3

